server/athenian/api/precompute/accounts.py

```diff
@@ -116,6 +129,9 @@ async def callback(context: PrecomputeContext):
                         reposet.owner_id,
                         status[1],
                     )
+                    track_failure()
+                else:
+                    track_success()
             track_time()
```

Comment:
I think we should move `track_time()` inside `finally:`, otherwise, we don't measure the time if there is a nasty system exception.